scripts/check_compliance.py

```diff
@@ -257,6 +257,10 @@ def parse_kconfig(self):
         os.environ["KCONFIG_WARN_UNDEF"] = "y"
 
         try:
+            # Note this will both print warnings to stderr _and_ return
+            # them: so some warnings might get printed
+            # twice. "warn_to_stderr=False" could unfortunately cause
+            # some (other) warnings to be printed never.
```

Comment:
Typo:

*to never be printed

Reply:
This is meant to be a stylistic effect, heard that in some movie(s) or song(s)... @dbkinder to the rescue?

Reply:
"printed once, printed twice, printed never"

Reply:
Yeah, sounds fine there at least.

Reply:
sigh.  "never" should not be at the end of a clause or sentence, and should be before the main verb of the sentence:  "He's never invited to dinner" and **not** "He's invited to dinner never".  So @ulfalizer is right:  ``cause some warnings to never be printed.``  (The "don't use split infinitives" rule isn't strictly followed these days: putting an adverb between ``to`` and the verb.)

Reply:
Maybe we're to blame for the split infinitives, going on [this video](https://www.youtube.com/watch?v=CDAU3TpunwM&t=14m10s). ;)

Reply:
Try to be a poet and get free English classes - what's not to like! Thanks @dbkinder.

(Now I hope no one looks at the commit message. github doesn't support reviewing and tracking commit messages; that's a good start)

Reply:
Love the video.  Thanks for sharing.